src/GraphQL/Types/Composite/ListGraphType.cs

```diff
@@ -2,7 +2,7 @@
 
 namespace GraphQL.Types
 {
-    public class ListGraphType<T> : ListGraphType
+    public class ListGraphType<T> : ListGraphType, IProvideResolvedType
```

Comment:
Why not on `ListGraphType` ?

Reply:
Same question for `NonNullGraphType`.

Reply:
@JKamsker  ?

Reply:
I mean to inherit `ListGraphType` and `NonNullGraphType` from `IProvideResolvedType`, not `ListGraphType<T>` and `NonNullGraphType<T>>`.